cleaner/cleaner.go

```diff
@@ -0,0 +1,38 @@
+package cleaner
+
+import (
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+)
+
+type t func() error
+
+var cleaners []t
+var inactive bool
+
+// Register a cleaner function. When a function is registered, the Signal watcher is started in a goroutine.
+func Register(f t) {
+	cleaners = append(cleaners, f)
+	if !inactive {
+		inactive = false
```

Comment:
Shouldn't that be `inactive = true` ? Inverting the meaning and using `active` would make it clearer.

Reply:
You are perfeclty right. Done.